app/views/computacenter/home/show.html.erb

```diff
@@ -13,12 +13,21 @@
         <%= govuk_button_link_to 'Download TechSource users', computacenter_user_ledger_path(format: :csv), classes: 'govuk-!-margin-top-4' %>
       </li>
     </ul>
+
     <ul class="govuk-list govuk-list--bullet">
       <li>
         <span class="govuk-!-margin-bottom-4">download a CSV file of Google Chromebook details for schools</span>
         <br /><br />
         <%= govuk_button_link_to 'Download Chromebook details', computacenter_chromebooks_path(format: :csv), classes: 'govuk-!-margin-top-4' %>
       </li>
     </ul>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        <span class="govuk-!-margin-bottom-4">update users that have TechSource accounts</span>
+        <br /><br />
+        <%= govuk_button_link_to 'Update TechSource users', computacenter_techsource_path, classes: 'govuk-!-margin-top-4' %>
+      </li>
+    </ul>
```

Comment:
This list with buttons layout is weird, I'll do a design pass on this once this PR is merged to make it a little more friendly.

Reply:
Your suggested changes have now been incorporated into this branch